Chapter1/AppCreationScripts/AppCreationScripts.md

```diff
@@ -1,35 +1,47 @@
-# Registering the sample apps with Microsoft identity platform and updating the configuration files using PowerShell scripts
+# Registering the sample apps with the Microsoft identity platform and updating the configuration files using PowerShell
 
 ## Overview
 
 ### Quick summary
 
-1. On Windows run PowerShell and navigate to the root of the cloned directory
+1. On Windows run PowerShell as **Administrator** and navigate to the root of the cloned directory
 1. In PowerShell run:
+
    ```PowerShell
    Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope Process -Force
    ```
+
 1. Run the script to create your Azure AD application and configure the code of the sample application accordingly. (Other ways of running the scripts are described below)
+
    ```PowerShell
    cd .\AppCreationScripts\
    .\Configure.ps1
    ```
+
 1. Open the Visual Studio solution and click start
 
 ### More details
 
 The following paragraphs:
 
-- [Present the scripts](#presentation-of-the-scripts) and explain their [usage patterns](#usage-pattern-for-tests-and-devops-scenarios) for test and DevOps scenarios.
-- Explain the [pre-requisites](#pre-requisites)
-- Explain [four ways of running the scripts](#four-ways-to-run-the-script):
-  - [Interactively](#option-1-interactive) to create the app in your home tenant
-  - [Passing credentials](#option-2-non-interactive) to create the app in your home tenant
-  - [Interactively in a specific tenant](#option-3-interactive-but-create-apps-in-a-specified-tenant)
-  - [Passing credentials in a specific tenant](#option-4-non-interactive-and-create-apps-in-a-specified-tenant)
-  - [Passing environment name, for Sovereign clouds](#running-the-script-on-azure-sovereign-clouds)
-
-## Goal of the scripts
+- [Registering the sample apps with the Microsoft identity platform and updating the configuration files using PowerShell](#Registering-the-sample-apps-with-the-Microsoft-identity-platform-and-updating-the-configuration-files-using-PowerShell)
```

Comment:
```suggestion
- [Registering the sample apps with the Microsoft identity platform and updating the configuration files using PowerShell](#registering-the-sample-apps-with-the-Microsoft-identity-platform-and-updating-the-configuration-files-using-PowerShell)
```
I don't think any of the paragraph links in this section work for the same reason (should start with lowercase)

Reply:
Aah, thanks for catching!